File: langchain/python/chapter01/b-chat.py
from langchain_openai.chat_models import ChatOpenAI
from langchain_core.messages import HumanMessage

model = ChatOpenAI(model="gpt-4o-mini")
prompt = [HumanMessage("What is the capital of France?")]

response = model.invoke(prompt)
print(response.content)

File: langchain/js/chapter01/b-chat.js
import { ChatOpenAI } from '@langchain/openai';
import { HumanMessage } from '@langchain/core/messages';

const model = new ChatOpenAI({ model: 'gpt-4o-mini' });
const prompt = [new HumanMessage('What is the capital of France?')];

const response = await model.invoke(prompt);
console.log(response.content);